client/analytics/report/customers/table.js

```diff
@@ -109,6 +108,8 @@ export default class CustomersReportTable extends Component {
 	}
 
 	getRowsContent( customers ) {
+		const { dateFormat } = window.wcSettings;
```

Comment:
You'll want to use `getSettings` as you mentioned in a previous conversation.
```js
import { getSettings } from '@woocommerce/wc-admin-settings';
```
`wcSettings` has been an abused dumping ground for settings, some of which are mutable. We're in the midst of a refactor to right some wrongs, but introduced `getSettings` as a way to access `wcSettings` in a more uniform manner. `dateFormat` is unlikely to be changed by wc-admin, so while this works, it'll be best to stick to convention.

Reply:
Ok, I'm much more comfortable with using `getSetting` instead of using a global directly. Thanks for this. It's changed now.